public/pages/DetectorConfig/containers/DetectorJobs.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import ContentPanel from '../../../components/ContentPanel/ContentPanel';
+import { EuiFlexGrid, EuiFlexItem, EuiFormRow, EuiText } from '@elastic/eui';
+import React from 'react';
+import { get, isEmpty } from 'lodash';
+import { ConfigCell } from '../../../components/ConfigCell';
+import { getHistoricalRangeString } from '../../../utils/utils';
+import { Detector } from '../../../models/interfaces';
+import { getDetectorStateDetails } from '../../DetectorDetail/utils/helpers';
+
+interface DetectorJobsProps {
+  detector: Detector;
+}
+
+export const DetectorJobs = (props: DetectorJobsProps) => {
+  const isHCDetector = !isEmpty(get(props, 'detector.categoryField', []));
+  const historicalEnabled = !isEmpty(get(props, 'detector.detectionDateRange'));
```

Comment:
Detector will not have `detectionDateRange` in unified flow. Should we remove this?

Reply:
I didn't change the [detector data model](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/public/models/interfaces.ts#L102) for a few reasons:
- simplicity in the changes, so all current places that are relying on this data model didn't need code changes yet (which is most pages/components)
- introducing more backward compatibility risk given the tight release timeline
- a lot of this was written before the backend design was finalized and I wanted to minimize breaking changes
Note that server-side code that parses the backend result has been updated to pull the [real-time detection task state & error message](https://github.com/ohltyler/anomaly-detection-dashboards-plugin-1/blob/single-flow/server/routes/ad.ts#L261).

I'm thinking now that the design has been finalized, can have a follow up task to fully refactor the frontend data models to be more consistent with the backend. Since this is more of an internal code clean up task, I think this can be tracked in an issue an fixed later. What do you think?

Reply:
Sure, sounds good, let's fix later.

Reply:
Created #28 to track.